src/plugins/intel_cpu/src/nodes/mkldnn_mvn_node.cpp

```diff
@@ -36,6 +36,62 @@ using namespace Xbyak;
 
 #define GET_OFF(field) offsetof(jit_mvn_call_args, field)
 
+namespace {
+struct MVNKey {
+    MKLDNNMVNNode::MVNAttrs mvnAttrs;
+    jit_mvn_config_params jcp;
+    mkldnn::primitive_attr attr;
+
+    size_t hash() const;
+    bool operator==(const MVNKey& rhs) const;
+};
+
+size_t MVNKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    seed = hash_combine(seed, std::get<0>(mvnAttrs.shape5D));
+    seed = hash_combine(seed, std::get<1>(mvnAttrs.shape5D));
+    seed = hash_combine(seed, std::get<2>(mvnAttrs.shape5D));
+    seed = hash_combine(seed, std::get<3>(mvnAttrs.shape5D));
+    seed = hash_combine(seed, std::get<4>(mvnAttrs.shape5D));
+    seed = hash_combine(seed, mvnAttrs.initAcrossChannels_);
+    seed = hash_combine(seed, mvnAttrs.execAcrossChannels_);
+    seed = hash_combine(seed, mvnAttrs.normalizeVariance_);
+    seed = hash_combine(seed, mvnAttrs.epsValue_);
+    seed = hash_combine(seed, mvnAttrs.epsMode_);
+    seed = hash_combine(seed, mvnAttrs.src_data_size);
+    seed = hash_combine(seed, mvnAttrs.dst_data_size);
+    seed = hash_combine(seed, mvnAttrs.is_nhwc);
+    seed = hash_combine(seed, jcp.planar_layout);
+    seed = hash_combine(seed, jcp.across_channels);
+    seed = hash_combine(seed, jcp.normalize_variance);
+    seed = hash_combine(seed, jcp.src_prc.getPrecVal());
+    seed = hash_combine(seed, jcp.dst_prc.getPrecVal());
+    seed = hash_combine(seed, jcp.src_data_size);
+    seed = hash_combine(seed, jcp.dst_data_size);
+    seed = hash_combine(seed, jcp.C);
+    seed = hash_combine(seed, jcp.D);
+    seed = hash_combine(seed, jcp.H);
+    seed = hash_combine(seed, jcp.W);
+    seed = hash_combine(seed, get_attr_hash(*attr.get()));
+    return seed;
+}
+
+bool MVNKey::operator==(const MVNKey& rhs) const {
+    bool retVal = true;
+    retVal = retVal && jcp.planar_layout == rhs.jcp.planar_layout && jcp.across_channels == rhs.jcp.across_channels &&
```

Comment:
The key comparor is not correct. mvnAttrs are not compared. This leads to wrong result when set dynamic shapes. Please change to following 
```
    bool retVal = true;
    retVal = retVal && mvnAttrs.shape5D == rhs.mvnAttrs.shape5D && mvnAttrs.initAcrossChannels_ == rhs.mvnAttrs.initAcrossChannels_ &&
             mvnAttrs.execAcrossChannels_ == rhs.mvnAttrs.execAcrossChannels_ && mvnAttrs.normalizeVariance_ == rhs.mvnAttrs.normalizeVariance_ &&
             mvnAttrs.epsValue_ == rhs.mvnAttrs.epsValue_ && mvnAttrs.epsMode_== rhs.mvnAttrs.epsMode_ &&
             mvnAttrs.src_data_size == rhs.mvnAttrs.src_data_size &&
             mvnAttrs.dst_data_size == rhs.mvnAttrs.src_data_size && mvnAttrs.is_nhwc == rhs.mvnAttrs.is_nhwc &&
             jcp.planar_layout == rhs.jcp.planar_layout && jcp.across_channels == rhs.jcp.across_channels &&
             jcp.normalize_variance == rhs.jcp.normalize_variance && jcp.src_prc == rhs.jcp.src_prc &&
             jcp.dst_prc == rhs.jcp.dst_prc && jcp.src_data_size == rhs.jcp.src_data_size &&
             jcp.dst_data_size == rhs.jcp.dst_data_size && jcp.C == rhs.jcp.C && jcp.D == rhs.jcp.D && jcp.H == rhs.jcp.H &&
             jcp.W == rhs.jcp.W;
    retVal = retVal && *attr.get() == *rhs.attr.get();
    return retVal;
```

Reply:
Done. Thanks Zhang Yi!